docs/deployment/linux/ubuntu.txt

```diff
@@ -0,0 +1,202 @@
+.. _ubuntu:
+
+=====================
+Run CrateDB on Ubuntu
+=====================
+
+CrateDB maintains packages for the following Ubuntu versions:
+
+- `Zesty Zapus`_ (17.04)
+- `Yakkety Yak`_ (16.10)
+- `Xenial Xerus`_ (15.04)
```

Comment:
Xenial Xerus is 16.04 LTS
http://releases.ubuntu.com/16.04/
must have been wrong in the old docs :-/

Reply:
fixed